{"uid":"b271348c56532eee","name":"Check Main Tab Functionality","fullName":"Check Main Tab Functionality","historyId":"54c37d45825518b63c108b26e89c05a0","time":{"start":1698705839317,"stop":1698705840265,"duration":948},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/app_03 (http://localhost:3001/main.js:45:25)\n    at __webpack_require__ (http://localhost:3001/main.js:130:41)\n    at initExternal (http://localhost:3001/main.js:407:28)\n    at __webpack_require__.I (http://localhost:3001/main.js:423:15)\n    at <unknown> (http://localhost:3001/main.js:536:47)\n    at webpack/sharing/consume/default/react/react?2849 (http://localhost:3001/main.js:590:72)\n    at <unknown> (http://localhost:3001/main.js:651:56)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.consumes (http://localhost:3001/main.js:634:36)\n    at <unknown> (http://localhost:3001/main.js:204:40)","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":4,"retriesStatusChange":true,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/app_03 (http://localhost:3001/main.js:45:25)\n    at __webpack_require__ (http://localhost:3001/main.js:130:41)\n    at initExternal (http://localhost:3001/main.js:407:28)\n    at __webpack_require__.I (http://localhost:3001/main.js:423:15)\n    at <unknown> (http://localhost:3001/main.js:536:47)\n    at webpack/sharing/consume/default/react/react?2849 (http://localhost:3001/main.js:590:72)\n    at <unknown> (http://localhost:3001/main.js:651:56)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.consumes (http://localhost:3001/main.js:634:36)\n    at <unknown> (http://localhost:3001/main.js:204:40)","steps":[{"name":"\"before each\" hook","time":{"start":1698705839317,"stop":1698705839916,"duration":599},"status":"passed","steps":[{"name":"visit http://localhost:3001","time":{"start":1698705839437,"stop":1698705839916,"duration":479},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true},{"name":"get button","time":{"start":1698705839931,"stop":1698705840264,"duration":333},"status":"failed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"uncaught exception ScriptExternalLoadError: Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03","time":{"start":1698705840264,"stop":1698705840264,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":0,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.runAllComprehensiveDemoReact16Tests.cy.ts"},{"name":"parentSuite","value":"Comprehemsive Demo React 16"},{"name":"suite","value":"Check is Comprehensive Demo App1 working and have elements"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"10da3b503c6fdeca","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698705838193,"stop":1698705839313,"duration":1120}},{"uid":"e03ba74848b08dca","status":"passed","time":{"start":1698705679813,"stop":1698705679813,"duration":0}},{"uid":"ba13f3d407df2e10","status":"passed","time":{"start":1698705676860,"stop":1698705676860,"duration":0}},{"uid":"7bf6543f4b38e06","status":"passed","time":{"start":1698705661848,"stop":1698705664430,"duration":2582}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":0,"skipped":5,"passed":11,"unknown":0,"total":19},"items":[{"uid":"83603172821a62c2","reportUrl":"https://module-federation.github.io/module-federation-examples/1468//#testresult/83603172821a62c2","status":"skipped","time":{"start":1698702011715,"stop":1698702011715,"duration":0}},{"uid":"2eecba9d85afd8db","reportUrl":"https://module-federation.github.io/module-federation-examples/1467//#testresult/2eecba9d85afd8db","status":"passed","time":{"start":1698696667679,"stop":1698696667680,"duration":1}},{"uid":"d77e7b8991ee32b3","reportUrl":"https://module-federation.github.io/module-federation-examples/1466//#testresult/d77e7b8991ee32b3","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3003/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/app_03\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698691623021,"stop":1698691623021,"duration":0}},{"uid":"790153605d4f1ffb","reportUrl":"https://module-federation.github.io/module-federation-examples/1459//#testresult/790153605d4f1ffb","status":"passed","time":{"start":1698655520196,"stop":1698655520196,"duration":0}},{"uid":"ed007c2965346c23","reportUrl":"https://module-federation.github.io/module-federation-examples/1458//#testresult/ed007c2965346c23","status":"passed","time":{"start":1698651645245,"stop":1698651645245,"duration":0}},{"uid":"78a44da5ce75b46b","reportUrl":"https://module-federation.github.io/module-federation-examples/1454//#testresult/78a44da5ce75b46b","status":"skipped","time":{"start":1698644383245,"stop":1698644383245,"duration":0}},{"uid":"925c4e65c3fab099","reportUrl":"https://module-federation.github.io/module-federation-examples/1453//#testresult/925c4e65c3fab099","status":"passed","time":{"start":1698640220310,"stop":1698640220310,"duration":0}},{"uid":"2e2de87c95c305ca","reportUrl":"https://module-federation.github.io/module-federation-examples/1446//#testresult/2e2de87c95c305ca","status":"skipped","time":{"start":1698629614212,"stop":1698629614213,"duration":1}},{"uid":"7b3f6c881ed27a02","reportUrl":"https://module-federation.github.io/module-federation-examples/1444//#testresult/7b3f6c881ed27a02","status":"skipped","time":{"start":1698618071649,"stop":1698618071650,"duration":1}},{"uid":"3ad308636e2d95b2","reportUrl":"https://module-federation.github.io/module-federation-examples/1443//#testresult/3ad308636e2d95b2","status":"skipped","time":{"start":1698616682211,"stop":1698616682211,"duration":0}},{"uid":"3ee6ae3375102cce","reportUrl":"https://module-federation.github.io/module-federation-examples/1442//#testresult/3ee6ae3375102cce","status":"passed","time":{"start":1698614018437,"stop":1698614018437,"duration":0}},{"uid":"352279c06c03c1f6","reportUrl":"https://module-federation.github.io/module-federation-examples/1426//#testresult/352279c06c03c1f6","status":"passed","time":{"start":1698562404425,"stop":1698562404425,"duration":0}},{"uid":"735c851027f31e37","reportUrl":"https://module-federation.github.io/module-federation-examples/1374//#testresult/735c851027f31e37","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Loading script failed.\n(error: http://localhost:3002/remoteEntry.js)\nwhile loading \"./Dialog\" from webpack/container/reference/app_02\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698528968518,"stop":1698528970106,"duration":1588}},{"uid":"c0a2cfd436edada6","reportUrl":"https://module-federation.github.io/module-federation-examples/897//#testresult/c0a2cfd436edada6","status":"passed","time":{"start":1682421368822,"stop":1682421368822,"duration":0}},{"uid":"660b46ee7f534e65","reportUrl":"https://module-federation.github.io/module-federation-examples/879//#testresult/660b46ee7f534e65","status":"passed","time":{"start":1682325254070,"stop":1682325254071,"duration":1}},{"uid":"b5c926da2098bc0","reportUrl":"https://module-federation.github.io/module-federation-examples/848//#testresult/b5c926da2098bc0","status":"passed","time":{"start":1682212105485,"stop":1682212105486,"duration":1}},{"uid":"65edb7ec4a2b0001","reportUrl":"https://module-federation.github.io/module-federation-examples/843//#testresult/65edb7ec4a2b0001","status":"passed","time":{"start":1682212105485,"stop":1682212105486,"duration":1}},{"uid":"e1c2584140bb7e82","reportUrl":"https://module-federation.github.io/module-federation-examples/831//#testresult/e1c2584140bb7e82","status":"passed","time":{"start":1682212105485,"stop":1682212105486,"duration":1}}]},"tags":[]},"source":"b271348c56532eee.json","parameterValues":[]}